src/sentry/api/serializers/models/sentry_function.py

```diff
@@ -6,7 +6,12 @@
 class SentryFunctionSerializer(Serializer):
     def serialize(self, obj, attrs, user):
         events = [event for event in obj.events]
-        env_variables = map(lambda x: {"name": x[0], "value": x[1]}, obj.env_variables.items())
+        env_variables = list(
+            map(
+                lambda env_variable: {"name": env_variable[0], "value": env_variable[1]},
+                obj.env_variables.items(),
+            )
+        )
```

Comment:
Originally, we only returned a map iterable object which is annoying if you're trying to test a serialized response. We should convert the iterable to a list before returning it